provider/datagen/src/baked_exporter.rs

```diff
@@ -462,10 +462,7 @@ impl BakedExporter {
                     &first_locale
                         .chars()
                         .flat_map(|ch| {
-                            if ch == AuxiliaryKeys::separator() as char {
-                                // Replace the aux key separator with double-underscore
-                                ['_'].into_iter().chain(Some('_'))
-                            } else if ch == '-' {
+                            if ch == '-' {
```

Comment:
the `flat_map` and `chain(None)` gymnastics can be removed

Reply:
Done